distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/csw/mock/FederatedCswMockServer.java

```diff
@@ -221,4 +213,13 @@ private String substituteSourceId(String document) {
     private String substituteHttpRoot(String document) {
         return document.replaceAll("\\$httpRoot\\$", httpRoot);
     }
+
+    private String getDefaultResponseResource(String resourceName) {
+        try {
+            return IOUtils.toString(FederatedCswMockServer.class.getResourceAsStream(
+                    "/" + resourceName));
+        } catch (IOException e) {
+            return null;
```

Comment:
Adding on to the previous response: for reusability purposes, we should allow the user to set this field if one is not present instead of being forced to include a default resource file.